openedx/features/course_card/admin.py

```diff
@@ -1,11 +1,19 @@
+"""
+Course Card and CourseRerunState Admin configurations
+"""
 from django.contrib import admin
 from django.forms import ModelForm, Select
+
 from course_action_state.models import CourseRerunState
 from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
 from openedx.features.course_card.models import CourseCard
 
 
 def get_parent_courses():
+    """
+    Getting All Courses detail from Course Reruns
+    Returns: tuple includes course_ids along with course display name concatenated with course_id
```

Comment:
Please add an empty space between the summary line and return arguments

From the docs:

```
def func(arg1, arg2):
    """
    Summary line.

    Extended description of function.

    Arguments:
        arg1 (int): Description of arg1
        arg2 (str): Description of arg2

    Returns:
        bool: Description of return value
    """
```